### PR TITLE
Moving v1beta1 kubectl examples to v1beta3

### PR DESCRIPTION
**Pull request by nikhiljindal**
Follow up of https://github.com/GoogleCloudPlatform/kubernetes/pull/6377
Am running e2e to verify that all is fine.

### CODE REVIEW

**Comment by nikhiljindal** [comment]
@bgrant0607

File: pkg/kubectl/cmd/get.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ $ kubectl get replicationController web
 $ kubectl get -o json pod web-pod-13je7
 
 // Return only the status value of the specified pod.
```
Comment (bgrant0607) [inline pkg/kubectl/cmd/get.go]: Please update the comment, also -- return only the phase

**Comment by bgrant0607** [comment]
Just one comment.
Let me know when e2e passes.

File: pkg/kubectl/cmd/get.go (hunk 1 above)
Comment (nikhiljindal) [inline pkg/kubectl/cmd/get.go]: Changed to "Return only the phase value of the specified pod."

**Comment by bgrant0607** [comment]
LGTM

**Comment by nikhiljindal** [comment]
e2e passed!!

**Pull request merged**